openupgrade_scripts/scripts/hr/16.0.1.1/post-migration.py

```diff
@@ -12,21 +12,25 @@ def _m2m_to_o2m_plan_activity_type_ids(env):
     between them then fill value for 'plan_id' at hr.plan.activity.type
     and after that ORM will do the rest for us
     """
-    openupgrade.logged_query(
-        env.cr,
+    env.cr.execute(
+        """
+        SELECT array_agg(hp.id) as hr_plan_ids, hpat.id as hr_plan_activity_type_id
+        FROM hr_plan hp JOIN hr_plan_hr_plan_activity_type_rel rel
+        ON hp.id = rel.hr_plan_id JOIN hr_plan_activity_type hpat
+        ON hpat.id = rel.hr_plan_activity_type_id
+        GROUP BY hpat.id
```

Comment:
@duong77476 như này là được mà nhỉ?
```suggestion
        SELECT array_agg(hr_plan_id) AS hr_plan_ids, hr_plan_activity_type_id
        FROM hr_plan_hr_plan_activity_type_rel
        GROUP BY hr_plan_activity_type_id
```

Reply:
vâng ạ